<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CS5490 Reader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Serial Port</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="portComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Temperature</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="temperatureValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Instantaneous Current</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="instCurrentValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Instantaneous Voltage</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="instVoltageValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Instantaneous Active Power</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="instActivePowerValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Active Power</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="activePowerValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RMS Current</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="rmsCurrentValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RMS Voltage</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="rmsVoltageValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Reactive Power</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="reactivePowerValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Peak Current</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="peakCurrentValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Peak Voltage</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="peakVoltageValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Apparent Power</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="apparentPowerValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Power Factor</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="powerFactorValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Total Active Power</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="totalActivePowerValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Total Apparent Power</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="totalApparentPowerValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Total Reactive Power</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="totalReactivePowerValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="textOutput"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="goButton">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
